code/modules/mob/living/basic/lavaland/bileworm/_bileworm.dm

```diff
@@ -1,4 +1,5 @@
 /mob/living/basic/mining/bileworm
+	SET_BASE_VISUAL_PIXEL(0, DEPTH_OFFSET / 2) // It's recessed into the ground
```

Comment:
in some places you manually touch DEPTH_OFFSET, in others you use a constant value. is that difference intentional?

Reply:
It's mostly "the ones I did earlier reference DEPTH_OFFSET and the later ones don't" because I realised that I was mostly just inserting random numbers, not modifications of depth_offset. I will probably change the other ones which reference the define into magic numbers also.